### PR TITLE
Concept blurbs and a review of their `about.md`s

### PR DESCRIPTION
**Pull request by angelikatyborska**
My approach is that concept blurbs should be like a very condensed concept introduction, describing the concept ("lists are... In Elixir, something..."), not the act of learning the concept ("You will learn about..."). According to Jeremy's message (https://exercism-team.slack.com/archives/CR91YFNG3/p1610128687115500?thread_ts=1610127582.114000&cid=CR91YFNG3), blurbs should be 10-70 _words_ long.

When reviewing concept about.md's, I was trying to:
- find broken links
- find grammar errors
- simplify long sentences
- add crucial missing information
- fix miscategorizing macros as functions, try to reference them them without using the word "macro"
- reformat list items as full sentences (capitalized, dot at the end)

### CODE REVIEW

File: concepts/cond/about.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-When we want to have branching code, we can use the [`cond/1`][cond] function:
```
Comment (angelikatyborska) [inline concepts/cond/about.md:1]: `cond` is a macro, but I think it's to early to explain the difference, so it's easier to refer to it as just "cond".

File: config.json [hunk 2]
```diff
@@ -1686,7 +1686,7 @@
       "uuid": "01211dc2-c8e1-4621-99a2-6aa66080839a",
       "slug": "multiple-clause-functions",
       "name": "Multiple Clause Functions",
-      "blurb": "TODO: add blurb for multiple-clause-functions concept"
+      "blurb": "In Elixir, a single function can have multiple clauses. This is achieved by pattern matching the function's arguments and by using guards."
```
Comment (angelikatyborska) [inline config.json:1689]: Unfortunately pattern-matching comes *after* multiple clause functions, but I think it's important to mention it in the blurb.

File: concepts/strings/about.md [hunk 3]
```diff
@@ -57,12 +57,6 @@ Some characters need to be [escaped][escape-characters] to be put in a string, e
 "\"A\" is the \#1st letter of the alphabet.\n"
 ```
 
-You can print a string to the console using [`IO.puts/2`][io-puts]:
-
-```elixir
-IO.puts("\"A\" is the \#1st letter of the alphabet.\n")
-```
-
```
Comment (angelikatyborska) [inline concepts/strings/about.md:65]: I think it's not necessary here, it's better if the students reads the `IO` concept if they want to know about printing strings.

File: concepts/tuples/about.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-Tuples are used commonly to group information informally. A common pattern through Elixir is to group function return values with a status.
+Tuples are used commonly to group information informally. A common pattern in Elixir is to group function return values with a status.
```
Comment (angelikatyborska) [inline concepts/tuples/about.md:1]: I wasn't sure if this was a grammar error or not... my hunch tells me it should be either "throughout" or "in".

File: concepts/enum/links.json [hunk 5]
```diff
@@ -6,5 +6,9 @@
   {
     "url": "https://hexdocs.pm/elixir/Enum.html",
     "description": "Documentation - Enum"
+  },
+  {
+    "url": "https://angelika.me/elixir-enum-cheatsheet/",
```
Comment (angelikatyborska) [inline concepts/enum/links.json:11]: Shameless plug! I hope it's allowed :)

**Review by neenjaw — APPROVED** [review]
Great work, these changes look good to me

**Comment by neenjaw** [comment]
ci just needs to be retriggered once your post-ve pr merges

**Comment by neenjaw** [comment]
actually I think you probably need to rebase off of `main` to get the updated ci workflows/etc

File: config.json [hunk 6]
```diff
@@ -1775,14 +1775,14 @@
...
-      "blurb": "TODO: add blurb for try-rescue concept"
+      "name": "Try/Rescue",
+      "blurb": "Elixir provides a construct for rescuing from errors using try/rescue."
     },
     {
       "uuid": "3beba34b-0675-48bf-9fe5-0e7e36eb6eea",
       "slug": "try-rescue-else-after",
-      "name": "Try Rescue Else After",
-      "blurb": "TODO: add blurb for try-rescue-else-after concept"
+      "name": "Try/Rescue/Else/After",
```
Comment (angelikatyborska) [inline config.json:1784]: I'm not sure if this spelling is better, but "Try Rescue Else After" reads really weird to me.

**Review by neenjaw — COMMENTED** [review]
still looks great

**Pull request merged**